src/robusta/cli/self_host.py

```diff
@@ -0,0 +1,180 @@
+import secrets
+import string
+import typer
+import yaml
+import json
+from typing import Any
+from pydantic import BaseModel
+from .backend_profile import BackendProfile
+import jwt as JWT
+
+
+def gen_secret(length: int) -> str:
+    return "".join(
+        secrets.choice(string.ascii_letters + string.digits) for _ in range(length)
+    )
+
+
+def write_values_files(
+    values_path: str,
+    backendconfig_path: str,
+    values: dict[str, Any],
+    backendProfile: BackendProfile,
+):
+    with open(values_path, "w") as output_file:
+        yaml.safe_dump(values, output_file, sort_keys=False)
+        typer.secho(
+            f"Saved configuration to {values_path} - save this file for future use!",
+            fg="red",
+        )
+
+    with open(backendconfig_path, "w") as output_file:
+        json.dump(
+            backendProfile.dict(exclude={"custom_profile"}), output_file, indent=1
+        )
+        typer.secho(
+            f"Saved configuration to {backendconfig_path} - save this file for future use!",
+            fg="red",
+        )
+
+
+class RobustaUI(BaseModel):
+    RELAY_HOST: str = ""
+    SUPABASE_URL: str = ""
+    SUPABASE_KEY: str = ""
+    service = {"nodePort": 30311}  # platform.domain
+
+    def __init__(self, domain: str, anon_key: str):
+        super().__init__(
+            RELAY_HOST=f"https://api.{domain}",
+            SUPABASE_URL=f"https://db.{domain}",
+            SUPABASE_KEY=anon_key,
+        )
+
+
+class RobustaRelay(BaseModel):
+    domain: str = ""
+    storePassword: str = ""
+    storeUser: str = "apiuser-robustarelay@robusta.dev"
+    storeUrl: str = ""
+    storeApiKey: str = ""  # anon key
+    slackClientId: str = "your-client-id"
+    slackClientSecret: str = "your-client-secret"
+    slackSigningSecret: str = "your-signing-secret"
+    syncActionAllowedOrigins: str = ""
+    provider: str = ""
+    apiNodePort: int = 30313  # api.domain
+    wsNodePort: int = 30314  # relay.domain
+
+    def __init__(self, domain: str, anon_key: str, provider: str, storePW: str):
+        super().__init__(
+            domain=domain,
+            storeUrl=f"https://db.{domain}",
+            syncActionAllowedOrigins=f"https://platform.{domain}",
+            storeApiKey=anon_key,
+            provider=provider,
+            storePassword=storePW,
+        )
+
+
+class SelfHostValues(BaseModel):
+    STATIC_IP_NAME: str = "robusta-platform-ip"
+    RELAY_PASSWORD: str = gen_secret(12)
+    RELAY_USER: str = "apiuser-robustarelay@robusta.dev"
+    DOMAIN: str = ""
+    PROVIDER: str = ""
+    # SUPABASE
+    JWT_SECRET: str = gen_secret(32)
+    JWT_EXPIRY: int = 3600
+    ANON_KEY: str = JWT.encode(
+        {"role": "anon", "iss": "supabase", "iat": 1661029200, "exp": 1818795600},
```

Comment:
I would comment what iss/iat/exp means
And I would put the values in constants with meaningful names. 
For example: `1_DAY_EXPIRATION = 1818795600`